source/using-ethereum-the-basics/ether.rst

```diff
@@ -5,27 +5,58 @@ Ether
 What is Ether?
 ================================================================================
 
+Ether is the base unit of currency used within Ethereum.  It is used to pay for
+computation within the EVM
+
 Denominations
 ================================================================================
 
-There are different denominations used in Ether and each denomination has its own unique name and value. Below is a list of different denomination names and how they are related to Ether.
+There are different denominations used in Ether and each denomination has its
+own unique name and value. Below is a list of different denomination names and
+how they are related to Ether.
 
-- 1 ether = 10^18 wei
-- 1 ether = 10^15 Kwei
-- 1 ether = 10^12 Mwei
-- 1 ether = 10^9 Gwei
-- 1 ether = 10^6 szabo
-- 1 ether = 10^3 finney
-- 1 ether = 1 ether
-- 1 ether = 1/(10^3) kether
-- 1 ether = 1/(10^6) Mether
-- 1 ether = 1/(10^9) Gether
-- 1 ether = 1/(10^12) Tether
++=====================+===========+===========================================+
+| unit                | wei value |                                           |
++=====================+===========+===========================================+
+| **wei**             | 1 wei     | 1                                         |
+| **Kwei (babbage)**  | 1e3 wei   | 1,000                                     |
+| **Mwei (lovelace)** | 1e6 wei   | 1,000,000                                 |
+| **Gwei (shannon)**  | 1e9 wei   | 1,000,000,000                             |
+| **szabo**           | 1e12 wei  | 1,000,000,000,000                         |
+| **finney**          | 1e15 wei  | 1,000,000,000,000,000                     |
+| **ether**           | 1e18 wei  | 1,000,000,000,000,000,000                 |
+| **kether**          | 1e21 wei  | 1,000,000,000,000,000,000,000             |
+| **Mether**          | 1e24 wei  | 1,000,000,000,000,000,000,000,000         |
+| **Gether**          | 1e27 wei  | 1,000,000,000,000,000,000,000,000,000     |
+| **Tether**          | 1e30 wei  | 1,000,000,000,000,000,000,000,000,000,000 |
++---------------------+-----------+-------------------------------------------+
 
 Getting Ether
 ================================================================================
 
+Version 0.3.9 (Beta 6) of the `Ethereum Wallet`_ introduced the ability to
+purchase ether via Shape Shift.
+
+You can also purchase Ether on any exchange which supports the currency.  As of
+2015-02-12 Poloniex was the exchange with the highest volume in Ethereum
+transactions.
```

Comment:
Well, the version string belongs to Mist, the Ethereum browser. The DApp wallet has [no version tags yet](https://github.com/ethereum/meteor-dapp-wallet/tags). There is a lot of confusion in different places about this. Not sure how to deal with that or what the plans for Homestead actually are.